docs/en/sql-reference/window-functions/index.md

```diff
@@ -23,6 +23,7 @@ ClickHouse supports the standard grammar for defining windows and window functio
 | `GROUPS` frame                                                                     | ❌                                                                                                                                                                               |
 | Calculating aggregate functions over a frame (`sum(value) over (order by time)`)   | ✅ (All aggregate functions are supported)                                                                                                                                                       |
 | `rank()`, `dense_rank()`, `row_number()`                                           | ✅                                                                                                                                                                                   |
+| `percent_rank()` | ✅  equal to `ifNull((rank() OVER(PARTITION BY x order by y) - 1) / nullif(count(1) OVER(PARTITION BY x) -1, 0), 0)`, but more efficient| 
```

Comment:
Isn't it the same as `(rank() OVER(PARTITION BY x order by y) - 1) / (count(1) OVER(PARTITION BY x) -1)` ?

Reply:
Yes, but need to consider the partitions with only one row.